src/router/index.js

```diff
@@ -12,6 +12,12 @@ const routes = [
     name: "About",
     component: () => import("../views/About.vue")
   },
+  {
+    path: "/campaigns/:address/edit",
+    name: "Edit Campaign",
+    component: () => import("../components/EditCampaignForm.vue"),
+    props: (route) => ({ ...route.params })
+  },
   {
     path: "/campaigns/:address",
```

Comment:
Nice routing! I think more specific routes should go below less specific routes to prevent mismatches. Just a small detail to consider.

Reply:
Right on, thanks for the feedback!